File: exercicios/Python/2143/2143.py
while 1:

    t = int(input())

    if t ==0:
        break

    for i in range(t):
        v = int(input())

        if v % 2 == 0:
            print(v * 2 - 2)
        else:
            print(v * 2 - 1)    
        

File: exercicios/JavaScript/2143/2143.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

while(true){
    var t = parseInt(lines.shift());

    if(t == 0){
        break;
    }

    for(let i=0; i<t;i++){
        var v = parseInt(lines.shift())

        if(v%2 == 0){
            console.log(v * 2 - 2)
        }else{
            console.log(v * 2 - 1)
        }
    }
}